integration/tests/managed/managed_nodegroup_test.go

```diff
@@ -549,6 +550,31 @@ var _ = Describe("(Integration) Create Managed Nodegroups", func() {
 			})
 		})
 
+		Context("and creating a nodegroup with containerd runtime", func() {
```

Comment:
the code you have added is for unmanaged nodegroups, this is a managed test.

we also need to do the opposite for each:
- pass the flag to the managed bootstrap code
- have integration tests for unmanaged nodegroups

Reply:
Ah, thanks